src/invidious/routes/video_playback.cr

```diff
@@ -278,8 +272,10 @@ module Invidious::Routes::VideoPlayback
       haltf env, status_code: 404
     end
 
-    url = URI.parse(url).request_target.not_nil! if local
-    url = "#{url}&title=#{title}" if title
+    if local
+      url = URI.parse(url).request_target.not_nil!
+      url += "&title=#{URI.encode_www_form(title, space_to_plus: false)}" if title
```

Comment:
Does encoding it twice mess anything up? It's also encoded here https://github.com/iv-org/invidious/pull/2922/files#diff-a86ced8aa99e3403588538decc008851d3d33dfadbb00d392d5ec8b4696f7df4R309-R312

Reply:
Yup, you're right. I fixed that.